pkg/create/templates/test.go

```diff
@@ -43,7 +43,19 @@ container_exists() {
 }
 
 container_ip() {
-  docker inspect --format="{{"{{"}} .NetworkSettings.IPAddress {{"}}"}}" $(cat $cid_file)
+  if (echo "$OSTYPE" | grep -qs 'darwin'); then
```

Comment:
@luiscom what I meant is that you can get the address of the container in the same way as before:
```
docker inspect --format="{{"{{"}} .NetworkSettings.IPAddress {{"}}"}}" $(cat $cid_file)
```

Simply change the way that it's curled on the Mac. Because you are starting another container to curl, the container should have access to the test container ip, even if your Mac doesn't.